lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/openturns/LinearModelStepwiseAlgorithm.hxx

```diff
@@ -49,12 +49,20 @@ public:
   LinearModelStepwiseAlgorithm();
 
   /** Parameters constructor */
-  LinearModelStepwiseAlgorithm(const Sample & inputSample,
-                               const Basis & basis,
-                               const Sample & outputSample,
-                               const Indices & minimalIndices,
+  LinearModelStepwiseAlgorithm(const Sample &inputSample,
+                               const Sample &outputSample,
+                               const Basis &basis,
+                               const Indices &minimalIndices,
                                const Direction direction = FORWARD,
-                               const Indices & startIndices = Indices());
+                               const Indices &startIndices = Indices());
+
+  /** Oldest constructor (for compatibility) */
+  LinearModelStepwiseAlgorithm(const Sample &inputSample,
+                               const Basis &basis,
+                               const Sample &outputSample,
+                               const Indices &minimalIndices,
+                               const Direction direction = FORWARD,
+                               const Indices &startIndices = Indices());
```

Comment:
its missing a notice in the API changes section of the changelog

Reply:
done